JupyterLab expects the share/jupyter/lab directory to be with the Python install.
By Simon Branford of the BEAR Software team at the University of Birmingham
--- jupyterlab/commands.py	2020-01-20 17:32:39.745748116 +0000
+++ jupyterlab/commands.py	2020-01-20 17:29:42.223438311 +0000
@@ -147,7 +147,7 @@
         return osp.abspath(os.environ['JUPYTERLAB_DIR'])
 
     # Use the default locations for data_files.
-    app_dir = pjoin(sys.prefix, 'share', 'jupyter', 'lab')
+    app_dir = pjoin(os.environ.get('EBROOTJUPYTERLAB', sys.prefix), 'share', 'jupyter', 'lab')
 
     # Check for a user level install.
     # Ensure that USER_BASE is defined
